modules/pam_mkhomedir/pam_mkhomedir.8.xml

```diff
@@ -46,6 +46,13 @@
       <filename>/etc/skel/</filename>) is used to copy default files
       and also sets a umask for the creation.
     </para>
+
+    <para condition="with_vendordir">
+      When default value <emphasis>/etc/skel</emphasis> is chosen,
+      <emphasis>%vendordir%/skel</emphasis> will also be used, and files
+      from default value will be preferred.
+    </para>
```

Comment:
The description is confusing.  It should explain that vendordir is used unless `skel=` option is specified.